packages/python/src/symbols/python-output-symbol.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Children,
+  OutputSymbol,
+  refkey,
+  Refkey,
+  useBinder,
+  useScope,
+} from "@alloy-js/core";
+
+/**
+ * Represents an 'exported' symbol from a .py file. Class, enum, interface etc.
+ */
+export interface PythonOutputSymbol extends OutputSymbol {
```

Comment:
Seems duplicated from the python-output-symbol.ts

Reply:
sorry I am ts noob, so I might be totally misunderstanding your comment: I thought I've only defined this interface in `python-output-symbol.ts` which is this file?